src/user_templates_api/templates/jupyter_lab/templates/hra_api_tutorial/template.txt

```diff
@@ -0,0 +1,1283 @@
+[
+  {
+    "attachments": {},
+    "cell_type": "markdown",
+    "id": "40dd5488",
+    "metadata": {},
+    "source": [
+      "## HRA-API Documentation"
```

Comment:
Please also add a description below this.

(Similar to the other template:  "This notebook shows how to use the different widgets provided by the hra_jupyter_widgets package.")